content/recipes/terminus.md

```diff
@@ -31,17 +31,23 @@ A health check represents a summary of **health indicators**. A health indicator
 - `MemoryHealthIndicator`
 - `DiskHealthIndicator`
 
-To get started with our first health check, we need to import the `TerminusModule` into our `AppModule`.
+To get started with our first health check, let's generate the `HealthModule` now using [Nest CLI](/cli/overview).
```

Comment:
We can't assume everyone uses CLI. We should skip this sentence and consider adding it as a hint

Reply:
Are you sure about that?
As Micael said earlier, the `nest g controller health` was there before any changes.

Should I still skip it?